packages/components/src/components/icon/icon-styles.ts

```diff
@@ -141,6 +160,11 @@ export const getComponentCss = (
               height: fontLineHeight,
               font: `${sizeMap[size]} ${fontFamily}`,
             }),
+        ...(isFlippableIcon(name) && {
```

Comment:
Wouldn't it be easier to maintain to flip all icons and exclude just logos and icons with numbers, e.g. "360"?